netty/src/test/java/io/grpc/netty/NettyServerHandlerTest.java

```diff
@@ -1280,4 +1280,18 @@ protected WriteQueue initWriteQueue() {
   protected void makeStream() throws Exception {
     createStream();
   }
+
+  @Override
+  protected AbstractStream stream()  {
+    if (stream == null) {
+      try {
+        makeStream();
+      } catch (Exception e) {
```

Comment:
Don't repackage the exception, just add `throws Exception` to the method signature.